backend/src/ILSpy.Host/Providers/CecilExtensions.cs

```diff
@@ -2,119 +2,231 @@
 // Copyright(c) 2017 ICSharpCode
 // Licensed under the MIT license. See the LICENSE file in the project root for more information.
 
+using System;
+using System.IO;
+using System.Reflection.PortableExecutable;
 using System.Text;
-using Mono.Cecil;
+using ICSharpCode.Decompiler.CSharp.OutputVisitor;
+using ICSharpCode.Decompiler.CSharp.Syntax;
+using ICSharpCode.Decompiler.Metadata;
+using ICSharpCode.Decompiler.TypeSystem;
 
 namespace ILSpy.Host.Providers
 {
     public static class CecilExtensions
     {
```

Comment:
Please rename the class name and the file name. `TypeSystemExtensions` or something similar?